pkg/dockerregistry/client.go

```diff
@@ -72,7 +72,7 @@ func (c *client) Connect(name string, allowInsecure bool) (Connection, error) {
 func normalizeRegistryName(name string) (*url.URL, error) {
 	prefix := name
 	if len(prefix) == 0 {
-		prefix = "index.docker.io"
+		prefix = "registry-1.docker.io"
```

Comment:
It doesn't, the v1 under registry-1 is broken.  This is going to have to be
a replacement at v2 check time.

On Aug 11, 2015, at 11:34 AM, Andy Goldstein <notifications@github.com>
wrote:

In pkg/dockerregistry/client.go
<https://github.com/openshift/origin/pull/4104#discussion_r36759319>:

> @@ -72,7 +72,7 @@ func (c *client) Connect(name string, allowInsecure bool) (Connection, error) {
>  func normalizeRegistryName(name string) (*url.URL, error) {
>  	prefix := name
>  	if len(prefix) == 0 {
> -		prefix = "index.docker.io"
> +		prefix = "registry-1.docker.io"

I wouldn't think this change would still allow v1 to function. Does it?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4104/files#r36759319>.